src/app/components/project-card/project-card.component.scss

```diff
@@ -1,18 +1,20 @@
 .in-progress {
-  padding: 10px;
+  padding: 1px;
   color: var(--ion-color-success); 
-  border-radius: 20px;
+  // border-radius: 20px;
   box-shadow: none;
+  width: 100%; /* label take up the full width of the column */
   }
   
   .not-started {
-    padding: 10px;
+    padding: 1px;
     color: var(--ion-color-danger);
```

Comment:
#4f4b48 use this color code for in progress status

Reply:
make one generic class 
   .status{
box-shadow: none;
    width: 20%;
    height: 50%;
    width: 100%;
padding: 1px;
}

.not-started {
color: #4f4b48 
}

.in-progress {
color :  #ffa500 
}
do like this